src/Card/Card.scss

```diff
@@ -206,6 +232,16 @@
   &.vertical {
     max-height: $card-image-vertical-max-height;
   }
+
+  .pgn__card-image-cap-loader {
+    .react-loading-skeleton {
+      margin-bottom: 1.25rem;
```

Comment:
Should we use an appropriate `$spacers` value here?

Reply:
Corrected